### ISSUE TITLE
AppCenter

### ISSUE THREAD

**Issue by jwinnie** [mank319/elementaryPlus#448]
This app should be published to the elementaryOS AppCenter.

**Comment by quassy** [comment]
You are right, but at the moment we are in dire need of active contributors to keep development going otherwise we would push an outdated program and icon pack.

**Comment by jwinnie** [comment]
@quassy How could I help update it?
